MultiplexReservationSystem/src/main/java/p/lodz/pl/multiplexreservationsystem/model/BookedSeat.java

```diff
@@ -0,0 +1,24 @@
+package p.lodz.pl.multiplexreservationsystem.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
```

Comment:
Importy z gwiazdką nie są poleceane, dlaczego?

Reply:
Importy z gwiazdką udostępniają wszytkie typy dostępne publicznie w danej przestrzeni nazw. Nie jest do dobre rozwiązanie, ponieważ dostarcza to klasie pewną redundancje/szum i drastycznie zwiększa zajmowaną pamięć pliku wykonywalnego/jara/assembly.

Reply:
Jest to oczywiście nie dopatrzenie z mojej strony. Miałem źle skonfigurowane IDE pod względem importów.

Reply:
Ja bym powiedział, że inny argument jest ważny. Szum bym nie powiedział, pamięć też jest tania (o ile faktycznie w skompilowanym pliku to by się rozwinęło do wszystkich klas- nie jestem przekonany, tam raczej jest używane tylko to co jest potrzebne). Natomiast:
- jeśli klasa o takiej samej nazwie jest w dwóch pakietach to mamy problem
- podczas robienia review nie jesteś w stanie stwierdzić skąd klasa pochodzi (czy z dobrych pakietów)